frontend/cvat/src/assets/engine/js/3rdparty/js.cookie.js

```diff
@@ -0,0 +1,165 @@
+/*!
```

Comment:
Is this file being used? Can it be removed?

Reply:
When I removed this file, I could no longer build my project